examples/CMakeLists.txt

```diff
@@ -1,13 +1,10 @@
+project(hipsycl-examples)
 
-find_program(SYCLCC NAMES syclcc CACHE STRING)
-if(SYCLCC MATCHES SYCLCC-NOTFOUND)
-  message(SEND_ERROR "hipSYCL syclcc compiler not found, exiting.")
-  return()
-endif()
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/../cmake)
 
-set(CMAKE_CXX_COMPILER ${SYCLCC})
-project(hipsycl-examples)
+find_package(hipSYCL CONFIG REQUIRED)
 
+set(CMAKE_CXX_COMPILER ${SYCLCC})
```

Comment:
This should not be necessary. `add_sycl_to_target` should take care of this.

Reply:
got it!